kimchi/src/circuits/lookup/index.rs

```diff
@@ -45,7 +45,7 @@ pub struct LookupSelectors<T> {
     /// LookupGate pattern lookup selector
     pub lookup_gate: Option<T>,
     /// RangeCheckGate pattern lookup selector
-    pub range_check_gate: Option<T>,
+    pub range_check: Option<T>,
```

Comment:
For consistency, make the name match the others ending with `_gate`

Reply:
I have made them uniform. For example, Xor not only names the Xor16 gate but also ChaCha0 and ChaCha1, but naming it XorGate could be misleading. I think the same type of situation may arise for other patterns as well.

Reply:
Yeah, disregard my comment.  For some reason I thought this was not the patterns.  Definitely we do not want a Gate suffix for the patterns :-)